src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/InstanceEventsControllerTests.cs

```diff
@@ -48,13 +49,13 @@ public InstanceEventsControllerTests(WebApplicationFactory<Startup> factory)
             public async void Post_UserHasToLowAuthLv_ReturnStatusForbidden()
             {
                 // Arrange
-                string requestUri = $"{BasePath}";
+                string requestUri = $"storage/api/v1/instances/1337/3c42ee2a-9464-42a8-a976-16eb926bd20a/events/";
```

Comment:
same instance for all tests, could we have instanceGuid as a property for the test class? Makes it easier to figure ou which tests data should be updated when new functionality is added to tests are to be expanded.

Reply:
I belive most test cases should have seperate instanceids.